test-utils/build.gradle

```diff
@@ -19,12 +19,17 @@ dependencies {
     }
 
     api group: 'junit', name: 'junit', version: '4.13.2'
+    api group: 'org.hamcrest', name: 'hamcrest-core', {
+        version {
+            strictly '2.2'
+        }
+    }
```

Comment:
This isn't something you need to fix in this PR, but I had some troubles running docker tests locally yesterday where the setup throws `java.lang.NoSuchMethodError: 'org.hamcrest.Matcher org.hamcrest.CoreMatchers.everyItem(org.hamcrest.Matcher)'`. When googling on it, it claimed it was due to a conflict between mockito and hamcrest version, but sadly enough your updates in the PR does not fix it even if I can navigate to the `everyItem` method in Hamcrest 2.2 😖 

I can circumvent is by commenting out //assumeNotNull(neo4jContainer);
from the Enterprise docker test locally but kind of weird and annoying problem
#endOfRant 😂

Reply:
I can take a look at that separately?